Orange/widgets/data/owfeatureconstructor.py

```diff
@@ -244,7 +244,7 @@ def setEditorData(self, data, domain):
     def editorData(self):
         values = self.valuesedit.text()
         values = re.split(r"(?<!\\),", values)
-        values = tuple(v.replace(r"\,", ",").strip() for v in values)
+        values = tuple(filter(None, [v.replace(r"\,", ",").strip() for v in values]))
```

Comment:
You can use generators instead of list comprehension, that is

values = tuple(filter(None, (v.replace(r"\,", ",").strip() for v in values)))